### DISCUSSION TITLE
brew tap NAME/versions can't find repository to clone

### DISCUSSION THREAD

**Discussion by aaelius** [Homebrew/discussions#2860] [Everyday usage]
I was trying `brew tap NAME/versions` over and over to install a formula from my organization and, the only way it actually worked was if I manually ran the git clone command but, instead of cloning with HTTP I would clone with SSH, it worked after that and I was able to install the formula. Has anyone ran into this before? or any idea why this would happen? I ran the same command about a month ago and it worked fine w/o workarounds.

I'm running
```
Homebrew 3.3.12-19-g51da118
Homebrew/homebrew-core (git revision 6333af44daf; last commit 2022-01-26)
Homebrew/homebrew-cask (git revision e4c1513a02; last commit 2022-01-26)
 ```

**Comment by SMillerDev** [comment]
Is the tap on github?

**Reply by aaelius** [reply]
yeh it is => [github.com/vht/versions](https://github.com/vht/homebrew-versions/)

**Reply by carlocab** [reply]
You need to specify the address of the tap since it doesn't follow the standard format for tap names.

See https://docs.brew.sh/Manpage#tap-options-userrepo-url

**Reply by aaelius** [reply]
oh that makes sense then, thanks!
